db/people.yml

```diff
@@ -61,6 +61,101 @@ achoudhry:
       status: undergrad
   name: Akshay Choudhry
   position: Undergraduate Researcher
+aday:
+  active: false
+  aka: []
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 0
+      institution: fix.me
+      degree: fix.me
+  employment:
+    - begin_year: 1999
+      position: Visiting Researcher
+      organization: msu
+      advisor: sbillinge
+      group: bg
+  name: A. R. Day
+adippel:
+  active: false
+  aka: []
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 2002
+      institution: RWTH Aachen
+      degree: Master degree
+  employment:
+    - begin_year: 2014
+      position: Visiting Researcher
+      organization: columbiau
+      advisor: sbillinge
+      group: bg
+  name: Ann-Christin Dippel
+aeberhadt:
+  active: false
+  aka: []
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 0
+      institution: fix.me
+      degree: fix.me
+  employment:
+    - begin_year: 1997
+      position: Visiting Researcher
+      organization: msu
+      advisor: sbillinge
+      group: bg
+  name: Ariane Eberhadt
```

Comment:
This one I can't find.  Let's put "unknown" for institution.  0 for the date is ok.  For the degree put PhD.  Let's use this pattern for all the ones we can't figure out.